src/integrations/TiktokAds/util.js

```diff
@@ -0,0 +1,68 @@
+import { constructPayload } from '../../utils/utils';
+import { PARTNER_NAME, trackMapping } from './constants';
+import { removeUndefinedAndNullValues } from '../../utils/commonUtils';
+
+const getContents = (message) => {
+    const contents = [];
+    const { properties } = message;
+    // eslint-disable-next-line camelcase
+    const { products, content_type, contentType } = properties;
+    if (products && Array.isArray(products) && products.length > 0) {
+        products.forEach((product) => {
+            const singleProduct = {
+                content_type:
+                    // eslint-disable-next-line camelcase
+                    product.contentType || contentType || product.content_type || content_type || 'product',
+                content_id: product.product_id,
+                content_category: product.category,
+                content_name: product.name,
+                price: product.price,
+                quantity: product.quantity,
+                description: product.description,
+            };
+            contents.push(removeUndefinedAndNullValues(singleProduct));
+        });
+    }
+    return contents;
+};
+
+const checkContentType = (contents, contentType) => {
```

Comment:
To avoid eslint
```suggestion
const checkContentType = (contents, contentType) => {
    let transformedContents = contents;
    if (Array.isArray(contents)) {
        transformedContents = contents.map( (content) => {
           const transformedContent = { ...content }
           if (!content.content_type) {
              transformedContent.content_type: contentType || 'product_group';
           }
           return transformedContent;
        });
    }
    return transformedContents;
};
```
Feel free to ignore(& resolve) this if this doesn't work-out(in terms of es-lint & functionality)

Reply:
Done